client/src/home/home.js

```diff
@@ -112,6 +124,7 @@ const HomeLayout = (props) => (
                 MISC2: query,
               });
             }}
+            include_gov={false}
```

Comment:
Maybe it felt redundant to the three big links that take you to gov infographics already? I think it'd be better to put it back in, since the search bar is above those and a user won't necessarily know those links go to the gov level unless they've already used them before.

Reply:
yeah agreed